definitions/ext-service/golden_metrics.yml

```diff
@@ -18,8 +18,9 @@ errorRate:
   title: Error rate (%)
   queries:
     opentelemetry:
-      select: (filter(count(*), where error.message IS NOT NULL) * 100) / count(*)
+      select: filter(count(*), WHERE otel.status_code = 'ERROR') / count(*)
       from: Span
+      where: span.kind LIKE 'server' OR span.kind LIKE 'consumer' OR kind LIKE 'server' OR kind LIKE 'consumer'
```

Comment:
Change this to `span.kind IN ('server', 'consumer') OR kind IN ('server', 'consumer')`

Reply:
This seemed to do the trick. Thank you.